src/server/cluster/cluster_slot_migration.h

```diff
@@ -37,12 +41,13 @@ class ClusterSlotMigration : ProtocolClient {
   std::error_code InitiateSlotsMigration();
 
  private:
+  Service& service_;
```

Comment:
where is service_ used?

Reply:
it is used in InitiateSlotsMigration() method to create ClusterShardMigration